onetl/connection/db_connection/iceberg/connection.py

```diff
@@ -68,19 +68,48 @@ class Iceberg(DBConnection):
             }
 
         This will be translated to:
+
         ``spark.sql.catalog.my_catalog = 'org.apache.iceberg.spark.SparkCatalog'``
+
         ``spark.sql.catalog.my_catalog.type = 'hadoop'``
+
         ``spark.sql.catalog.my_catalog.warehouse = 'file:///path/to/warehouse'``
 
     Examples
     --------
 
+    **REST catalog + S3 warehouse**
```

Comment:
This requires SparkS3.get_packages() and also to specify S3 options within `extra` (bucket, access_key, secret_key and so on)